docs/apache-airflow-providers-dbt-cloud/operators.rst

```diff
@@ -108,6 +115,7 @@ Use the :class:`~airflow.providers.dbt.cloud.sensors.dbt.DbtCloudJobRunAsyncSens
 status of a dbt Cloud job run asynchronously. This sensor will free up the worker slots since
 polling for job status happens on the Airflow triggerer, leading to efficient utilization
 of resources within Airflow.
+:class:`~airflow.providers.dbt.cloud.sensors.dbt.DbtCloudJobRunAsyncSensor` is deprecated and will be removed in a future release. Please use :class:`~airflow.providers.dbt.cloud.sensors.dbt.DbtCloudJobRunSensor` and use the deferrable mode in that operator.
```

Comment:
Would you mind including this note in a `.. note::` directive? Let's remove the code snippet example below too since that example shows a deprecated sensor.

Reply:
sure! I just update it. I'm not sure whether we can remove the code snippet. If my memory serves me right, it won't pass the CI.